dot/parachain/network-bridge/receiver.go

```diff
@@ -43,6 +44,7 @@ type NetworkBridgeReceiver struct {
 
 	BlockState *state.BlockState
 	Keystore   keystore.Keystore
+	sync       Sync
```

Comment:
This isn't being set yet, when the `NetworkBridgeReceiver` is constructed, right?

Reply:
done